content/cpp/concepts/pointers/pointers.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: "Pointers"
+Subjects:
+  - "Computer Science"
+  - "Game Development"
+Tags: 
+  - "Pointers"
+CatalogContent:
+  - "learn-c-plus-plus"
+  - "paths/computer-science"
+---
+
+A pointer is a speacial type variable that stores address of another variable in it. 
```

Comment:
Replace "speacial" with "special". 

Insert "of" in between "type" and "variable".

Insert "the" in between "stores" and "address".